corehq/ex-submodules/toggle/shortcuts.py

```diff
@@ -3,13 +3,12 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
 
-from .models import Toggle
```

Comment:
why was it necessary to remove this module-level import?

Reply:
I believe that this file gets imported in different places before the models have been initialized and raise `django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet`

Reply:
ugh 🤕